### PR TITLE
Create GitHub Action, and clean up.

### PR DESCRIPTION
**Pull request by hamishmack**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      shell: bash
+      run: |
+        curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o jq
+        chmod +x jq
+    - name: Download and import closure
+      shell: bash
+      run: |
+        TOKEN=$(curl --silent https://ghcr.io/token\?scope\=repository:input-output-hk/devx:pull | jq -r .token)
+
+        BLOB=$(curl \
```
Comment (angerman) [inline action.yaml:44]:
```suggestion
        # Fetch Docker image containing our nix store closure.
        BLOB=$(curl \
```

File: action.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          --silent \
+          --request 'GET' \
+          --header "Authorization: Bearer $TOKEN" \
+          --header "Accept: application/vnd.docker.distribution.manifest.list.v2+json" \
+          --header "Accept: application/vnd.docker.distribution.manifest.v2+json" \
+          --header "Accept: application/vnd.oci.image.manifest.v1+json" \
+          "https://ghcr.io/v2/input-output-hk/devx/manifests/${{ inputs.platform }}.${{ inputs.compiler-nix-name }}${{ inputs.target-platform }}${{ inputs.minimal && '-minimal' || '' }}${{ inputs.iog && '-iog' || '' }}-env" | tee manifest.json | jq -r '.layers[0].digest')
+
+        curl \
```
Comment (angerman) [inline action.yaml:53]: # Download the docker image layer that contains our nix store closure, and import it.

Comment (angerman) [inline action.yaml:53]:
```suggestion
        # Download the docker image layer that contains our nix store closure, and import it.
        curl \
```

File: action.yaml (hunk 1 above)
Comment (hamishmack) [reply action.yaml]: I've added this comment to the script file

File: action.yaml (hunk 2 above)
Comment (hamishmack) [reply action.yaml]: I've added this comment to the script file

**Pull request merged**